test/runAllTests.cmd

```diff
@@ -0,0 +1,8 @@
+%DNX_HOME%\runtimes\dnx-clr-win-x86.1.0.0-beta4\bin\dnx.exe .\Microsoft.ApplicationInsights.AspNet.Tests test
```

Comment:
I added this for my convenience. If someone has time to do something fancy please do.